ios/Classes/Bindings/FilePresentationServiceBinding.swift

```diff
@@ -15,6 +15,11 @@ private enum EventKeys: String, CaseIterable {
 
 class FilePresentationServiceBinding: Binding {
     
+    override init(name: String, registrar: FlutterPluginRegistrar) {
+        super.init(name: name, registrar: registrar)
+        VoxeetSDK.shared.filePresentation.delegate = self
```

Comment:
I think we can put it in `onInit()`

`    override func onInit() {
        VoxeetSDK.shared.filePresentation.delegate = self
    }`

Reply:
Done